lib/pyszn/injection.py

```diff
@@ -179,7 +182,9 @@ def parse_attribute_injection(
 
                 # Nodes
                 if 'nodes' in modifier:
-                    for node in _expand_nodes(parsed_topology, modifier['nodes']):
+                    for node in _expand_nodes(
```

Comment:
why this change?

Reply:
flake8.
I ran tests and flake8 complained, so I fixed this code. I am guessing CIT does not run the tests